build/release/teamcity-support.sh

```diff
@@ -15,3 +13,51 @@ tc_start_block() {
 tc_end_block() {
   echo "##teamcity[blockClosed name='$1']"
 }
+
+log_into_gcloud() {
+  if [[ "${google_credentials}" ]]; then
+    echo "${google_credentials}" > .google-credentials.json
+    gcloud auth activate-service-account --key-file=.google-credentials.json
+  else
+    echo 'warning: `google_credentials` not set' >&2
+  fi
+}
+
+docker_login_with_google() {
+  # https://cloud.google.com/container-registry/docs/advanced-authentication#json-key
+  echo "${google_credentials}" | docker login -u _json_key --password-stdin "https://${gcr_hostname}"
```

Comment:
@jlinder curious you are using `docker login` instead of `gcloud auth configure-docker`

Reply:
I used `docker login` because that is what `gcloud` [directed me to do](https://cloud.google.com/container-registry/docs/advanced-authentication#json-key) when trying to set up `gcloud auth configure-docker`. I think it's a new change for gcloud (like within the last month or so).